metagraph/src/graph/alignment/dbg_aligner.cpp

```diff
@@ -441,19 +457,22 @@ ::extend_chain(std::string_view query,
     assert(!best.empty());
 
     if (best.get_end_clipping()) {
+        logger->trace("Extending back");
```

Comment:
Would this not generate too much stuff? Maybe it's better to print it as `DEBUG(...)`?